docs/direct-debits/How-To-Request-Payments.md

```diff
@@ -0,0 +1,65 @@
+---
+tags: [Direct Debits]
+---
+
+# How To Request Payments
+
+## Creating a Mandate
+Before you can make any payments you have to first create a Mandate. To create a new mandate you must have already created a party and bank account. A mandate is only an instruction to give authority to take payments. As such you can create a mandate without creating a portfolio or account first.
+
+The bank account number and sort code used to create the mandate must be able to pass a Modulus check. To test on sandbox  you should use `55779911` as the account number and `20-00-00` as the sort code.
+
+Once you've met the prerequisites you can request a new mandate using the <a href="/docs/api/docs/openapi/api.yaml/paths/~1direct-debits~1mandates/post">Create Mandate API</a>.
+
+When you create a mandate it starts with a `Pending` status. You cannot create payments until it has transitioned into an 'Active' status. It's also at this stage you would display the success page referred to in the [UI Guidelines](./Api-Access.md#page-3---success-screen). You can monitor this using the returned mandate Id against the <a href="/docs/api/docs/openapi/api.yaml/paths/~1direct-debits~1mandates~1%7BmandateId%7D/get">Get Mandate API</a>.
+
+The end user will receive an email, usually within an hour, of the mandate being create. This email have the mandate PDF attached to it.
+
+> It's still possible for a mandate to fail after it's reached the Active state. If this happens, any payments created against the mandate will failed as well.
```

Comment:
Just as a styling not, if you want different colours for these (say as a warning) then you can use this:

https://meta.stoplight.io/docs/studio/ZG9jOjg0-stoplight-flavored-markdown#block-types

Reply:
Thanks for the suggestion, I did see those, but I think simply highlighting them is enough.